### ISSUE TITLE
Helm's --cleanup-on-fail option makes troubleshooting a failed deploy impossible

### ISSUE THREAD

**Issue by mrzarquon** [gitpod-io/gitpod#9838]
### Bug description

When a deploy fails, or fails too fast for helm, it proceeds to delete all pods associated with the installation.

https://github.com/gitpod-io/gitpod/blob/main/install/kots/manifests/gitpod-installer-job.yaml#L276

Editing and removing this line from my failed install and doing a redeploy allowed it to be successful, eventually, it just took longer for the migration pod to complete it's job, and then the services came back online.

### Steps to reproduce

Perform an installation with gitpod 2022.4.2, using in cluster services, make a configuration change, and perform a redeploy. If there is any job failure, it appears to delete the pods it installed.

**Comment by corneliusludmann** [comment]
Independent (?) of this, would it make sense to increase the 5 minute timeout to give the DB migration more time?

**Comment by MrSimonEmms** [comment]
I don't think we need to do anything on this. Since `2022.6.0`, we have Fluent Bit installed on clusters and installer logs are copied to a support bundle. If there are any issues with finding out what happened in any ephemeral pod (installer, workspaces, imagebuild), the are now in support bundles.

@mrzarquon please reopen if this isn't working for you in support
